deployments/examples/gang/README.md

```diff
@@ -0,0 +1,28 @@
+<!--
+* Licensed to the Apache Software Foundation (ASF) under one
+* or more contributor license agreements.  See the NOTICE file
+* distributed with this work for additional information
+* regarding copyright ownership.  The ASF licenses this file
+* to you under the Apache License, Version 2.0 (the
+* "License"); you may not use this file except in compliance
+* with the License.  You may obtain a copy of the License at
+*
+*      http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+-->
+
+# Gang in application level
+
+To run the `gangDeploy.sh` script with parameter:
+
+```shell script
+./cmd/gangDeploy.sh <job number> <pod amount> <gang member> <task run time(sec)>
```

Comment:
Where is a `podAmount` and a `gangMember`? 
To simplify this, I think we can only have 1 parameter, `gangMember`.

Reply:
So, it should be `./cmd/gangDeploy.sh <job number> <gang member> <task run time(sec)>` , right?

Reply:
yes, I meant to say: let's remove `<pod amount>` : )